### PR TITLE
Feature/appv1

### PR DESCRIPTION
**Pull request by tnyandoro**
Weather app

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Invalid Code Review Request :x: 

__Hi @tnyandoro, I commend you for the great effort put into this project. However, I have to mark the project as invalid due to the following reasons:__

- [ ] Somall linter che checks were not successful: to consider this project ready for review, you need to ensure that all linter checks are running/passing successfully. 
- [ ] Your pull request does not have a meaningful title and a short summary. You need to summarize what you've done in this project in your pull request message.

**Friendly reminder:** You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
Do not forget to apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

_**Do ensure that the issues above are fixed before requesting another review.**_

* * *
**Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * * 

**_You can reach out to me on Twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray:
Nice work on this project. I like the Apple design for the UI. 
I was able to search for weather based on different locations and the background image changed based on the data received. Well done!
Let's make some crucial changes before merging this PR.
- [ ] Add a toggle button for changing temperature unit between Fahrenheit and Celsius
- [ ] Remove comments from `index.js` and `weather.js` files. Comments make your code look ugly.
- [ ] The `index.js` and `weather.js` files both have similar code. Remove one file since it's redundant.
- [ ] Add error handling to the `promises` logic.
- [ ] Update readme instructions. Check my comments for suggestions.
- [ ] Make your PR more descriptive


### Martin Afani
#### Slack: whiz25

**Comment by tnyandoro** [comment]
This is a Web app that displays weather for different cities in the world per request. 
Removed comments from index.js and weather.js files. 
 Removed  index.js and weather.js files both have similar code. Remove one file since it's redundant.
 Resolved error handling to the logic of the promise.
 Updated readme instructions. 
 PR more descriptive

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Changes Required ❌

# 👋 team

#### Good job on addressing some of the issues raised in the previous review, duplicate file was removed and adding error handling promises🤗. However:

* [ ] We still have commented code in the `index.js` file, let us remove all commented code, as requested in the previous review.

* [ ] Let us add a toggle feature, ideally a button that would enable users of the app view temperature in degree Celsius or Fahrenheit when clicked as requested in the previous review.

* [ ] I could not launch the application on my local machine by following the steps outlined in the `readme`, let make sure the steps in the `readme` to get the app to start is really what will cause the application to be launched in the local environment. What happens after the command `npm run build`?

#### That is it, kindly make the changes and submit a re-review😎

Happy coding

Addo Forison

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    <script
+      src="https://kit.fontawesome.com/fddcccfd38.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- Stylesheets -->
+    <link rel="stylesheet" href="./style.css" />
+
+    <!-- Bootstrap -->
+    <!-- <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous"> -->
```
Comment (Forison) [inline dist/index.html:20]: remove this line if not needed

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      .then((response) => response.json())
+      .then((data) => this.displayWeather(data));
+  },
+
+  displayWeather(data) {
+    const { name } = data;
+    const { icon, description } = data.weather[0];
+    const { temp, humidity } = data.main;
+    const { speed } = data.wind;
+    //  const { all } = data.clouds;
```
Comment (Forison) [inline src/index.js]: Remove commented code

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @tnyandoro :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by tnyandoro** [comment]
Final commit

**Pull request closed without merging**